src/state/user/hooks.tsx

```diff
@@ -137,6 +138,23 @@ export function useUserSlippageTolerance(): Percent | 'auto' {
   )
 }
 
+export function useUserHideClosedPositions(): [boolean, (newHideClosedPositions: boolean) => void] {
+  const dispatch = useDispatch<AppDispatch>()
+
+  const hideClosedPositions = useSelector<AppState, AppState['user']['userHideClosedPositions']>(
+    (state) => state.user.userHideClosedPositions
```

Comment:
fyi this will be undefined if state is loaded from `localStorage`, if you want to add something to the 'updateVersion' handler to set it to false or just treat undefined like false in this hook

Reply:
I think that should be okay. Tempted to leave as-is for consistency with the other boolean flags in state.